.github/CODEOWNERS

```diff
@@ -97,3 +97,9 @@
 /src/js_native_api* @nodejs/n-api
 /doc/guides/adding-new-napi-api.md @nodejs/n-api
 /doc/api/n-api.md @nodejs/n-api
+
+# gyp
+
+/**/*.gyp @nodejs/gyp
```

Comment:
Will `*.gyp` work ?

Reply:
No idea 😅

The best thing to do is to test with the action and with the algorithm on github-bot since there might be some discrepancies between them (but hopefully there is no discrepancy)